### PR TITLE
Added seq checking for assembly ref passing to GB upload.

### PR DESCRIPTION
**Pull request by jkbaumohl**

### CODE REVIEW

File: lib/GenomeFileUtil/core/GenbankToGenome.py [hunk 1]
```diff
@@ -356,11 +357,22 @@ def _save_assembly(self, genbank_file, params):
...
-            assembly_contig_ids = set(ret['data']['contigs'].keys())
-            unmatched_ids = set(self.contig_seq.keys()) - assembly_contig_ids
-            if unmatched_ids:
-                raise ValueError("The genbank file contains contigs which are not present in the "
-                                 "supplied assembly".format(", ".join(unmatched_ids)))
+            unmatched_ids = list()
+            unmatched_ids_md5s = list()
+            print "SELF CONTIG SEQ KEYS: " + str(self.contig_seq.keys())
+            print "SELF CONTIG SEQ : " + str(self.contig_seq)
+            print "PREV UPLOADED CONTIGS : " + str(ret['data']['contigs'])
```
Comment (JamesJeffryes) [inline lib/GenomeFileUtil/core/GenbankToGenome.py]: I think we can drop these print statements

Comment (jkbaumohl) [reply lib/GenomeFileUtil/core/GenbankToGenome.py]: Yeah my Bad.  Forgot cleanup

Comment (jkbaumohl) [reply lib/GenomeFileUtil/core/GenbankToGenome.py]: Ok done. Do I need to do a new PR?

**Comment by JamesJeffryes** [comment]
Nope, now that I think of it, should this be a "supplemental test" not a "core" test?

**Comment by JamesJeffryes** [comment]
If we could demonstrate this with a shorter genbank file that would be great. (Reason being that the more test data in the repo, the slower tests and registrations run) too but I wouldn't hold this up for that.

**Comment by jkbaumohl** [comment]
I can remove the annotations from the newest file added. I need 3 files in total. 2 were already part of the repo.  The new one was for sequence changes.
I can do this stuff after lunch.

**Comment by jkbaumohl** [comment]
Ok done

**Pull request merged**